frontend/src/useFlow.ts

```diff
@@ -205,35 +228,72 @@ const createSources = async (name: string, originalLanguage: string, translatedL
     original_source_id: originalSource.id,
     dictionary_id: dictionaryId || undefined,
     dictionary_timestamp: dictionaryTimestamp || undefined,
+    properties: {
+      is_original: false,
+    },
   });
   console.log(translatedSource);
 
+  // Create source entries for all additional sources
+  const additionalSourceIds: number[] = [];
+  for (const additionalSource of additionalSources) {
+    const additionalSourceEntry = await postSource({
+      name: `${baseName}`,
+      language: additionalSource.language,
+      properties: {
+        is_original: false,
+      },
+    });
+    additionalSourceIds.push(additionalSourceEntry.id);
+    console.log('Created additional source:', additionalSourceEntry);
+  }
+
+  // Create links between all sources and the translated source
+  await postSourceOriginLinks(originalSource.id, additionalSourceIds, translatedSource.id);
```

Comment:
It will be much cleaner/simpler if the API will\ receive a list of tuples which are the links themselves:
[
  {original_source_id_1, translated_source_id_1},
  {original_source_id_2, translated_source_id_2},
  ...
]